packages/fiori/src/Timeline.ts

```diff
@@ -105,14 +117,22 @@ class Timeline extends UI5Element {
 	}
 
 	_onfocusin(e: FocusEvent) {
-		const target = e.target as TimelineItem;
+		let target = e.target as ITimelineItem | ToggleButton;
```

Comment:
why do you need heere `ITimelineItem | ToggleButton`

Reply:
because when initially focused, the target is the group as a whole, so in order to achieve functional focusing behavior and visually at all, we shall cast it to the button. Once the button is focused, the next or previous targets are correct (timeline-items).